make.html

```diff
@@ -19,7 +19,13 @@ <h2>What is redirectatob?</h2>
     </div>
     <script>
       function update() {
-        var u = 'http://redirectatob.github.io/index.html#' + btoa(document.getElementById('url').value);
+        var u = 'http://redirectatob.github.io/index.html#';
+        var r = document.getElementById('url').value;
+        try {
+          u += btoa(r)
+        } catch (e) {
+          u += btoa("!" + encodeURIComponent(r))
```

Comment:
Nit: this line and `u += btoa(r)` should have semicolons.

(I don't use semicolons in my JS code anymore, but I'm not bothering updating the code style of this repository for now.)